### PR TITLE
FEATURE: [FE] Homepage feature

### PR DESCRIPTION
**Pull request by sayyidyofa**
I am working on a frontend feature to make a homepage

Changes:

 - Added LandingPage page.
 - Added import semantic css lib to index.tsx
 - Changed App.tsx to only render LandingPage
 - Changed Register.tsx and Profile.tsx to comply with TypeScript linting rule for new files

Request for code review @fannyhasbi @abdarafi

### CODE REVIEW

File: frontend/src/pages/home/Profile.tsx [hunk 1]
```diff
@@ -0,0 +1 @@
+export {}
```
Comment (fannyhasbi) [inline frontend/src/pages/home/Profile.tsx]: Delete the file if it doesn't containt anything

File: frontend/src/pages/auth/Register.tsx [hunk 2]
```diff
@@ -0,0 +1 @@
+export {}
```
Comment (fannyhasbi) [inline frontend/src/pages/auth/Register.tsx]: Delete the file if it doesn't containt anything

**Review by fannyhasbi — CHANGES_REQUESTED** [review]
Please change these

**Comment by fannyhasbi** [comment]
Please consider using TSLint if possible, because it helps much in formatting and checking

File: frontend/src/pages/home/Profile.tsx (hunk 1 above)
Comment (sayyidyofa) [reply frontend/src/pages/home/Profile.tsx]: Done in 297b95af686333995dfbb1642990bc3e4eced889

File: frontend/src/pages/auth/Register.tsx (hunk 2 above)
Comment (sayyidyofa) [reply frontend/src/pages/auth/Register.tsx]: Done in 297b95af686333995dfbb1642990bc3e4eced889

**Comment by fannyhasbi** [comment]
Do you want to use TSLint in this PR? @sayyidyofa

**Comment by sayyidyofa** [comment]
I will use TSLint in my next PR @fannyhasbi

**Review by fannyhasbi — APPROVED** [review]

**Comment by fannyhasbi** [comment]
Approved. You can continue merge this PR and close it

**Pull request merged**